src/applications/disability-benefits/all-claims/pages/secondaryUploadSources.js

```diff
@@ -2,6 +2,11 @@ import { ancillaryFormUploadUi } from '../utils';
 
 import { uploadDescription } from '../content/fileUploadDescriptions';
 import { ptsd781aNameTitle } from '../content/ptsdClassification';
+// import fullSchema from 'vets-json-schema/dist/21-526EZ-ALLCLAIMS-schema.json';
```

Comment:
I'm assuming we're going to get the schema PR in first and then add this back?

Reply:
Yes, we shouldn't be using the local schema file. Adding this to the page schema makes sense to me, but if we have to add them to the all-claims schema, we'll need a `vets-json-schema` PR as well.